include/mbedtls/pkcs7.h

```diff
@@ -202,15 +218,26 @@ int mbedtls_pkcs7_parse_der( mbedtls_pkcs7 *pkcs7, const unsigned char *buf,
  * \note           This function internally calculates the hash on the supplied
  *                 plain data for signature verification.
  *
- * \return         A negative error code on failure.
+ * \return         0 if the signatures match, or a negative error code on failure.
```

Comment:
We normally talk about the signature (singular) verifying
```suggestion
 * \return         0 if the signature verifies, or a negative error code on failure.
```